scripts/run-travis-tests.sh

```diff
@@ -3,7 +3,7 @@ set -ev
 npm run lint
 npm run test:coverage
 npm run test:karma
-coveralls < ./test/coverage/lcov.info
+cat ./test/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Revert https://github.com/callemall/material-ui/pull/4941 as seems to be linked to some flakiness with the coverage fails.